.github/workflows/docker.yaml

```diff
@@ -0,0 +1,151 @@
+name: Build, test, and push Docker Images
+on:
+  push:
+    branches:
+      - "master"
+    paths:
+      - ".github/workflows/*"
+
+      - "images/**"
+      - "!images/*/README.md"
+      - "tests/**"
+      - "!tests/README.md"
+      - "requirements-dev.txt"
+  pull_request:
+    paths:
+      - ".github/workflows/*"
+
+      - "images/**"
+      - "!images/*/README.md"
+      - "tests/**"
+      - "!tests/README.md"
+      - "requirements-dev.txt"
+
+jobs:
+  vars:
+    runs-on: ubuntu-latest
+    outputs:
+      REGISTRY_NAME: "k8scc01covidacr.azurecr.io"
```

Comment:
Does this have to be an output for this `vars` job? Since it's just a static value, can't it be an env var for the workflow? (Like how it used to be done in build_push,yaml)
I don't think this would change anything on how the workflow runs, it would just be for readability.

Reply:
I think the reason was to keep all the set up in the same location. And since branch name couldn't be set in env, I put everything in vars.

Do you still think I should move it to an env?

Reply:
Ok it does make sense to keep all the setup in the same location, even if it's maybe a little weird to just output a static value. 
That is okay, we can keep it as is.